CRM/Member/Task.php

```diff
@@ -102,6 +102,16 @@ public static function tasks() {
           'class' => 'CRM_Member_Form_Task_PDFLetter',
           'result' => FALSE,
         ),
+        self::SAVE_SEARCH => array(
```

Comment:
@mattwire this is the part of the change I'm focussed on

Reply:
@eileenmcnaughton Yes, I'm pretty sure it does.  When I added it to CRM/Member/Task I was being cautious but ideally we'd move this into CRM_Core_Task - not sure I've time to thoroughly test that change though at the moment but the groundwork should all be in place for that to be done in the future.